src/QMCHamiltonians/NonLocalECPComponent.h

```diff
@@ -140,9 +141,9 @@ class NonLocalECPComponent : public QMCTraits
 
   void resize_warrays(int n, int m, int l);
 
-  void randomize_grid(RandomGenerator& myRNG);
+  void randomize_grid(const TensorType& rmat);
```

Comment:
This function no more handles randomization. Hence a rename and also adopt naming convention. rotateQuadratureGrid?